.github/workflows/stale-issues-pr.yml

```diff
@@ -0,0 +1,28 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 * * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@72afbce2b0dbd1d903bb142cebe2d15dc307ae57
+        with:
+          stale-issue-message: 'This issue has been automatically marked as stale because it has not had recent activity in the last 90 days. It will be closed in 7 days. Thank you for your contributions.'
+          stale-issue-label: 'stale'
+          only-issue-labels: 'type-bug'
+          exempt-issue-labels: 'type-security, type-pinned, feature-request, awaiting-metamask'
+          stale-pr-message: 'This PR has been automatically marked as stale because it has not had recent activity in the last 60 days. It will be closed in 7 days. Thank you for your contributions.'
+          stale-pr-label: 'stale'
+          exempt-pr-labels: 'work-in-progress'
+          close-issue-message: 'This issue was closed because there has been no follow up activity in the last 7 days. If you feel this was closed in error please reopen and provide evidence on the latest release of the extension. Thank you for your contributions.'
+          close-pr-message: 'This PR was closed because there has been no follow up activity in the last 7 days. Thank you for your contributions.'
+          days-before-issue-stale: 90
+          days-before-pr-stale: 60
+          days-before-issue-close: 7
```

Comment:
What if we start with an initial phase where issues are closed after a longer period. Perhaps 45 days. This will give us time to look at all of the stale issues. Then, we can reduce the time after the first 45 day cycle.

Reply:
great idea Dan, I'll add that in

Reply:
We could do a similar thing with PRs to start, perhaps 14 days in that case.